src/store/index.js

```diff
@@ -49,6 +53,7 @@ export default function (storage) {
               'apps',
               'rpcUrl',
               'mobile.keystore',
+              'mobile.derivedKey',
```

Comment:
`derivedKey` is derived from user password and used for decryption of master key, it shouldn't be stored in secure or local storage

Reply:
@davidyuk understood!